src/app/containers/Figure/__snapshots__/index.test.jsx.snap

```diff
@@ -547,18 +547,18 @@ exports[`Figure should render an image with caption text with inline link 1`] =
 }
 
 .c3 {
-  font-size: 0.9375rem;
+  font-size: 0.9375em;
   line-height: 1.125rem;
   background-color: #D5D0CD;
   color: #404040;
-  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-family: ReithSansNewsRegular,Helvetica,Arial,sans-serif;
```

Comment:
Similar to the `rem` -> `em` changes this is a regression in the package, so will also change back to `ReithSans`